src/renderer/components/screens/MainScreen.tsx

```diff
@@ -118,6 +118,11 @@ export default function MainScreen() {
       ]
       if (lastChatId) {
         selectChat(Number(lastChatId))
+      } else {
+        // Welcome the user with the "Device Messages" chat in case no chat was
+        // selected previously (for example when they've just joined Delta Chat
+        // for the first time)
+        selectChat(C.DC_CONTACT_ID_DEVICE)
```

Comment:
this does not work, DC_CONTACT_ID_DEVICE is a contact-id, not a chat-id.

there is no api to get the device-chat-id directly; however, as there are only two chats on startup, just iterate over them and check for [`dc_chat_is_device_talk()`](https://c.delta.chat/classdc__chat__t.html#a560a4ccfa512cad1ecb066ac275b76b8) resp. the corresponding jsonrpc call (i think, this is good enough, the effort to create and maintain a new api is probably higher)